workflow/scripts/build_egon_data.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
```

Comment:
Some style suggestions, keep the same order in the script as for example [`build_existing_chp_de`](https://github.com/PyPSA/pypsa-ariadne/blob/4aab9c834e116a1dfcfab2a8335a5a2b75a586d1/workflow/scripts/build_existing_chp_de.py#L1-L9)

1. docstring
2. import packages
3. dictionaries, defined lists...
4. functions
5. then the main code